django_firebase_auth/views.py

```diff
@@ -70,7 +82,7 @@ def _get_or_create_user(email: str) -> Optional[AbstractBaseUser]:
         if not CREATE_USER_IF_NOT_EXISTS:
             return None
 
-    user = UserModel.objects.create_user(email=email, is_active=True)
+    user = UserModel.objects.create_user(username=email, email=email, is_active=True)
```

Comment:
`create_user` requires a username at least with the default `django.contrib.auth`.
@mpnowacki-reef did you have some custom auth module in mind?
Using `email` as a username may be fine for some use-cases, but maybe not all. Should I make this behavior configurable? I.e. use either email or firebase ID, or something else based on settings?

Reply:
yeah, the one project we use this app in currently uses a custom UserModel with no `username` field.

Reply:
although I think this additional property will just be ignored

Reply:
now that I think of it, as a next extension of this app, a function for creating users should be configurable

Reply:
no, it breaks our current app. I think for now let's leave it without `username` and later on let's add that configurable hook for creating users.

Reply:
Ok, removed the `username` parameter.